docs/docs/overview/how-does-testkube-work.md

```diff
@@ -0,0 +1,5 @@
+### How do I install Testkube?
+
+To install Testkube, you'll need to install Testkube CLI (which is currently available on MacOS, Linux, and Windows) and install Testkube's API in your cluster using Helm.
```

Comment:
`using Helm` - using Helm directly isn't required - `testkube install` can be used instead. 
`and install Testkube's API in your cluster` - maybe `and install Testkube cluster-side components in your cluster`?